robottelo/ui/puppetclasses.py

```diff
@@ -58,3 +58,60 @@ def delete(self, name, really):
 
         self.delete_entity(name, really, locators["puppetclass.select_name"],
                            locators['puppetclass.delete'])
+
+    def import_scap_client_puppet_classes(self):
+        """Imports puppet-foreman_scap_client puppet classes."""
+        Navigator(self.browser).go_to_puppet_classes()
+        self.wait_for_ajax()
+        self.find_element(locators['puppetclass.import']).click()
+        # Checking if the scap client puppet classes are already imported
+        if self.wait_until_element(
+                locators['puppetclass.environment_default_check'],
+                5,
```

Comment:
It takes more that enough time to wait for element to appear. So decreased it so that execution will be faster. It doesnt harms the test.